server/src/main/java/org/apache/druid/guice/FirehoseModule.java

```diff
@@ -58,7 +59,8 @@ public void configure(Binder binder)
                 new NamedType(CombiningFirehoseFactory.class, "combining"),
                 new NamedType(FixedCountFirehoseFactory.class, "fixedCount"),
                 new NamedType(SqlFirehoseFactory.class, "sql"),
-                new NamedType(InlineFirehoseFactory.class, "inline")
+                new NamedType(InlineFirehoseFactory.class, "inline"),
+                new NamedType(SqlInputSource.class, "sql")
```

Comment:
can this be part of new module as InputSource seems to replacement for firehose related interfaces ?

Reply:
Thanks. Extracted it to a separate module.

Reply:
Thanks, however I would have just called it `InputSourceModule` and made it available to all processes by adding it in `makeInjectorWithModules` of `Initialization.java` similar to `FirehoseModule`.

Reply:
I can refactor the module name, but I believe this module wouldn't be needed in the broker, historical and router processes since its specific to indexing.

Reply:
ok